css/index.css

```diff
@@ -1,3 +1,484 @@
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
 
+/* HTML5 display-role reset for older browsers */
+
+article, aside, details, figcaption, figure, footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+
+body {
+	line-height: 1;
+}
+
+ol, ul {
+	list-style: none;
+}
+
+blockquote, q {
+	quotes: none;
+}
+
+blockquote:before, blockquote:after, q:before, q:after {
+	content: '';
+	content: none;
+}
+
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+
+* {
+	box-sizing: border-box;
+}
+
+html, body {
+	height: 100%;
+	font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+	font-family: 'Bangers', cursive;
+	letter-spacing: 1px;
+	margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+
+@import url('https://fonts.googleapis.com/css?family=Yanone+Kaffeesatz');
+
+@media(max-width: 800px) {
+	body {
+		overflow-y: scroll;
+		overflow-x: hidden;
+	}
+}
+
+.container {
+	width: 800px;
+	margin: 0 auto;
+}
+
+.contain-service {
+	width: 800px;
+	margin: 0 auto;
+}
+
+@media(max-width: 500px) {
+	body {
+		overflow-y: scroll;
+		overflow-x: hidden;
+	}
+	
+	.container {
+		width: 500px;
+	}
+
+	.contain-service {
+		width: 500px;
+	}
+}
+
+nav {
+	margin: 10px;
+	text-align: center;
+	background: white;
+	padding-bottom: 5px;
+	z-index: 1;
+	width: 800px;
+}
+
+nav ul {
+	display: flex;
+	justify-content: space-evenly;
+}
+
+nav a {
+	font-family: 'Yanone Kaffeesatz', sans-serif;
+	color: black;
+	text-decoration: none;
+	padding: 4px 20px;
+	text-align: center;
+	border-radius: 50px;
+	transition: background 1s ease, color .5s ease, border-radius .5s ease;
+}
+
+nav a:hover {
+	color: white;
+	background: black;
+	border-radius: 0;
+	box-shadow: 0 8px 6px -6px gray;
+	cursor: pointer;
+}
+
+@media(max-width: 800px) {
+	nav {
+		margin-top: 110px;
+		display: flex;
+		text-align: center;
+		width: 800px;
+	}
+	nav a {
+		padding: 5px 37px;
+	}
+
+	nav a:hover {
+		color: inherit;
+		background: inherit;
+		box-shadow: 0 0px 0px 0px transparent;
+	}
+}
+
+@media(max-width: 500px) {
+	nav {
+		width: 100%;
+		height: 400px;
+		padding: 0;
+		margin: 5px auto;
+		position: relative;
+		top: 72px;
+	}
+
+	nav ul {
+		display: inline-block;
+		width: 100%;
+	}
+
+	nav a {
+		border-radius: 0px;
+		line-height: 60px;
+		font-size: 19px;
+	}
+
+	nav li {
+		border-top: 1px solid black;
+	}
+}
+
+.logo {
+	vertical-align: text-bottom;
+	width: 150px;
+	padding-left: 10px;
+}
+
+
+@media(max-width: 800px) {
+	.logo {
+		position: relative;
+		top: -78px;
+		left: -476px;
+		width: 195px;
+	}
+}
+
+@media(max-width: 500px) {
+	.logo {
+		position: relative;
+		top: -423px;
+		left: -5px;
+	}
+}
+
+header {
+	text-align: center;
```

Comment:
Good use here. Some were using `<br>` in the HTML but we want to avoid that when possible.